broken.js

```diff
@@ -0,0 +1,2 @@
+fodinfoidsifaf
+fdiaf
```

Comment:
Expected an assignment or function call and instead saw an expression.<br>Missing semicolon.<br>'fdiaf' is not defined.